### DISCUSSION TITLE
Checking for updates?

### DISCUSSION THREAD

**Discussion by rkrug** [posit-dev/positron#3990] [Q&A] [answered]
Hi

am I right, that at the moment there is no way to check for a new version of positron apart from manually via the website?

Thanks.

**Comment by jmcphers** [comment, marked as answer]
You're correct. This is something we hope to address before Positron's first stable release. You can follow the work here: https://github.com/posit-dev/positron/issues/1837

**Comment by rkrug** [comment]
Thanks
